tests/e2e/system_test/mon/test_mon_crash_recovery_scenario.py

```diff
@@ -0,0 +1,112 @@
+import logging
+import pytest
+from threading import Thread
+
+from ocs_ci.ocs import constants
+from ocs_ci.helpers.helpers import modify_deployment_replica_count
+from ocs_ci.ocs.resources.deployment import get_mon_deployments
+from ocs_ci.ocs.resources.pvc import get_pvc_objs
+from ocs_ci.utility.utils import ceph_health_check
+from ocs_ci.ocs.resources.pod import get_ceph_tools_pod
+from ocs_ci.ocs.resources.storage_cluster import ceph_mon_dump
+from ocs_ci.framework.pytest_customization.marks import tier4a
+
+
+log = logging.getLogger(__name__)
+
+
+@tier4a
+@pytest.mark.polarionid("OCS-4942")
+class TestMonCrashRecoveryScenario:
+    def teardown_resources(self):
+        """Teardown function to scale deployments back to 1 replica."""
+        for dep in ["ocs-operator", "rook-ceph-operator"]:
+            modify_deployment_replica_count(dep, 1)
+
+    def test_mon_crash_recovery_scenario(self, pvc_factory, pod_factory, request):
+        """
+        Verifies system behavior when a crash occurs in the mon-x deployment.
+
+        Steps:
+            1. Start the IO workload in the background.
+            2. Scale down the deployments of the operators and of the mon-x.
+            3. Delete the Deployment of rook-ceph-mon-x and pvc rook-ceph-mon-x
+            4. Scale up the operators to replicas = 1
+            5. Verify 'ceph mon dump' command is working.
+            6. Check for the any crash has generated.
+        """
+
+        mon_obj = get_mon_deployments()[0]
+        mon_name = mon_obj.name
+        mon_pvc = mon_obj.data["metadata"]["labels"]["pvc_name"]
+        mon_pvc_obj = get_pvc_objs([mon_pvc])[0]
+
+        # Step1:  Creating Workload On the Cluster.
```

Comment:
you can directly call the pod_factory that will take care of pvc creation

Reply:
Done